seed/ts-sdk/exhaustive/package-path/package.json

```diff
@@ -0,0 +1,70 @@
+{
+    "name": "@fern/exhaustive",
+    "version": "0.0.1",
+    "private": false,
+    "repository": "https://github.com/exhaustive/fern",
+    "type": "commonjs",
+    "main": "./dist/cjs/index.js",
+    "module": "./dist/esm/index.mjs",
+    "types": "./dist/cjs/index.d.ts",
+    "exports": {
+        ".": {
+            "types": "./dist/cjs/index.d.ts",
+            "import": {
+                "types": "./dist/esm/index.d.mts",
+                "default": "./dist/esm/index.mjs"
+            },
+            "require": {
+                "types": "./dist/cjs/index.d.ts",
+                "default": "./dist/cjs/index.js"
+            },
+            "default": "./dist/cjs/index.js"
+        },
+        "./package.json": "./package.json"
+    },
+    "files": [
+        "dist",
+        "reference.md"
+    ],
+    "scripts": {
+        "format": "prettier . --write --ignore-unknown",
+        "build": "yarn build:cjs && yarn build:esm",
+        "build:cjs": "tsc --project ./tsconfig.cjs.json",
+        "build:esm": "tsc --project ./tsconfig.esm.json && node scripts/rename-to-esm-files.js dist/esm",
+        "test": "jest tests/unit --passWithNoTests",
+        "test:wire": "jest tests/wire --passWithNoTests",
+        "wire:test": "yarn test:wire"
```

Comment:
This is debatable, so would love others opinion on this.
I think (not sure), we should limit the tests the subpackage path.

Reply:
```text
yarn test
yarn run v1.22.22
warning package.json: No license field
$ jest tests/unit --passWithNoTests
● Validation Error:

  Module <rootDir>/tests/mock-server/setup.ts in the setupFilesAfterEnv option was not found.
         <rootDir> is: /Users/niels/fern/seed/ts-sdk/exhaustive/package-path

  Configuration Documentation:
  https://jestjs.io/docs/configuration

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```